Components/BlockImage/index.twig

```diff
@@ -1,22 +1,23 @@
 <div class="flyntComponent flyntComponent--{{ options.size }} {{ options.theme }} {{ noComponentSpacing == true ? '' : 'componentSpacing' }}" is="flynt-block-image">
   <div class="container{% if options.size != 'sizeFull' %} centerMaxWidthContainer{% endif %}">
     <figure class="figure">
-      {% set gif = image.post_mime_type == 'image/gif' %}
-      <img class="figure-image {{ not gif ? 'lazyload' }}"
-        src="{{ not gif ? image.src|resizeDynamic(800) : image.src }}"
-        {% if not gif %}
-          srcset="{{ placeholderImage(800, 800 / image.aspect) }}"
-          data-srcset="
-            {{ image.src|resizeDynamic(1920) }} 1920w,
-            {{ image.src|resizeDynamic(1440) }} 1440w,
-            {{ image.src|resizeDynamic(1320) }} 1320w,
-            {{ image.src|resizeDynamic(1024) }} 1024w,
-            {{ image.src|resizeDynamic(800) }} 800w,
-            {{ image.src|resizeDynamic(600) }} 600w,
-            {{ image.src|resizeDynamic(375) }} 375w"
-          data-sizes="auto"
-        {% endif %}
-        alt="{{ image.alt|e }}">
+      {% if image.post_mime_type == 'image/svg+xml' or image.post_mime_type == 'image/gif' %}
+        <img src="{{ image.src }}" alt="{{ image.alt|e }}">
```

Comment:
Should this also get a lazyload class? I thought it might be just missing in the original code snippet as well.